ARCHITECTURE.md

```diff
@@ -0,0 +1,56 @@
+# PrestoDB: Mission and Architecture
+
+## Introduction
+
+PrestoDB is an open-source distributed SQL query engine designed for fast analytic queries against data sources of all sizes, ranging from gigabytes to petabytes. It was originally developed by Facebook and later made available to the broader community. It is governed under the auspices of the Presto Foundation, a member of the Linux Foundation.
+
+## Mission
+
+### For users
+
+The primary mission of PrestoDB is to enable efficient and high-speed data processing for analytics and batch at scale. It aims to provide a single, unified query system that can access and process data stored in various formats and storage systems. Key aspects of its mission include:
+
+1. **High Performance:** Presto strives to exceed data warehouse performance in data lakes. To achieve this, Presto works to achieve vertical integration over modular portions of the stack.
+2. **Scalability:** Presto is used at extremely large data lakes across the world. As such, it aims for high degrees of scalability.
```

Comment:
As I mentioned in the row ID RFC, just because Presto only reads columnar data over data lakes, doesn't mean Presto doesn't operate over data lakes.

There is a federation aspect to Presto and there's a data lake query aspect to Presto.  We do both, but we focus (in terms of where we put the majority of our effort) on the latter.  By calling out specifically data lakes, I am trying to call out this focus in the doc.